modules/nf-core/merfin/hist/tests/main.nf.test

```diff
@@ -0,0 +1,579 @@
+                    ],[
+                    [ id:'test_fragilis', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path + '/genomics/prokaryotes/bacteroides_fragilis/genome/genome.fna.gz', checkIfExists: true)
+                    ])
+                    input[1] = Channel.value(6)
+                    """
+                }
+            }
+
+            run("MERYL_HISTOGRAM") {
+                script "../../../meryl/histogram/main.nf"
+                config "./modules.config"
+                tag    "meryl/histogram"
+                process {
+                    """
+                    input[0] = MERYL_COUNT_READS.out.meryl_db
+                    input[1] = Channel.value(6)
+                    """
+                }
+            }
+
+            run("GENOMESCOPE2") {
+                script "../../../genomescope2/main.nf"
+                config "./modules.config"
+                tag    "genomescope2"
+                process {
+                    """
+                    input[0] = MERYL_HISTOGRAM.out.hist
+                    """
+                }
+            }
+
+
+            
+            
+        }
+
+        when {
+            process {
+                """
+                Channel.from([
+                    [ id:'test_sars', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path + '/genomics/sarscov2/genome/genome.fasta', checkIfExists: true)
+                    ],[
+                    [ id:'test_fragilis', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path + '/genomics/prokaryotes/bacteroides_fragilis/genome/genome.fna.gz', checkIfExists: true)
+                    ]).set{ input_zero_ch }
+
+                input_zero_ch.join( MERYL_COUNT_READS.out.meryl_db )
+                             .join( MERYL_COUNT_ASM.out.meryl_db )
+                             .join( GENOMESCOPE2.out.lookup_table )
+                             .multiMap{ meta, asm, meryl_db, asm_meryl_db, lookup_table ->
+                    asm_ch: [meta, asm]
+                    meryl_ch: [meta, meryl_db]
+                    asm_meryl_ch: [asm_meryl_db]
+                    lk_ch: [lookup_table]
+                }.set{ multimap_inputs_ch }
+                
+                input[0] = multimap_inputs_ch.asm_ch
+                input[1] = multimap_inputs_ch.meryl_ch
+                input[2] = multimap_inputs_ch.lk_ch
+                input[3] = MERYL_COUNT_ASM.out.meryl_db.map{ meta, meryl_db -> meryl_db }
+                input[4] = 10
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.get(0), process.out.get(2)).match() },
+                // The following asserts that the output stdout_log generated by the module is complete
```

Comment:
This comment is confusing. You can probably remove it as I assume it's a copy-paste from above?

Reply:
It's true.. I forgot to remove this comment before. Done.